llvm/bindings/ocaml/llvm/llvm.ml

```diff
@@ -446,7 +446,7 @@ external module_context : llmodule -> llcontext = "llvm_get_module_context"
 external get_module_identifier : llmodule -> string
                                = "llvm_get_module_identifier"
 
-external set_module_identifer : llmodule -> string -> unit
+external set_module_identifier : llmodule -> string -> unit
                               = "llvm_set_module_identifier"
```

Comment:
nit: would be nice to preserve the vertical alignment of `:` and `=`

Reply:
oops! I miss the important detail. Thank you for saying, and i will take care more.
And sorry for asking because i'm not familiar with contributing. Am i have to fix about comment? or generally commenter already fixed it?
Thank you, and i hope you have a nice day!